Library/Homebrew/utils/inflection.rb

```diff
@@ -0,0 +1,18 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Utils
+  # Inflection utility methods, as a lightweight alternative to `ActiveSupport::Inflector``.
+  #
+  # @api private
+  module Inflection
+    extend T::Sig
+
+    # Combines `stem`` with the `singular`` or `plural` suffix based on `count`.
+    sig { params(count: Integer, stem: String, plural: String, singular: String).returns(String) }
+    def self.number(count, stem, plural = "s", singular = "")
```

Comment:
Thoughts on making `plural` and `singular` keyword rags? Could even abbreviate to `pl` and `sg` if line length is a concern. I think it will be much easier to use the methods this way since remembering the order won't be necessary.

```suggestion
    def self.number(count, stem, plural: "s", singular: "")
```

I also like the `pluralize` naming, and if we go with that I think it makes sense to have the stem come first. So ultimately:

```suggestion
    def self.pluralize(stem, count, plural: "s", singular: "")
```